taskcluster/ffios_taskgraph/transforms/bitrise.py

```diff
@@ -102,21 +102,21 @@ def add_bitrise_command(config, tasks):
         yield task
 
 
-@transforms.add
-def add_screenshot_checks_command(config, tasks):
-    for task in tasks:
-        commands = task["run"]["commands"]
+# @transforms.add
```

Comment:
Nit: Let's add a comment to remember to comment this back in.

Reply:
oh, right, I only added it to the test 🤦‍♀️